src/mame/video/k051960.cpp

```diff
@@ -319,21 +328,18 @@ void k051960_device::k051937_w(offs_t offset, u8 data)
 		/* bit 3 = flip screen */
 		m_spriteflip = data & 0x08;
 
-		/* bit 4 used by Devastators and TMNT, unknown */
+		/* bit 4 used by Devastators and TMNT, to protect sprite RAM from corruption during updates ? */
 
 		/* bit 5 = enable gfx ROM reading */
 		m_readroms = data & 0x20;
 		//logerror("%s: write %02x to 051937 address %x\n", m_maincpu->pc(), data, offset);
 	}
 	else if (offset == 1)
 	{
-		//popmessage("%04x: write %02x to 051937 address %x", m_maincpu->pc(), data, offset);
-		// Chequered Flag uses this bit to enable background palette dimming
-		// TODO: use a callback here for now, pending further investigation over this bit
-		m_vreg_contrast_handler(BIT(data,0));
```

Comment:
This removal sounds like this callback in chqflag.cpp is now orphanized.
https://github.com/mamedev/mame/blob/ef98b0220d4de4535d9bf10dfde536c19afc3517/src/mame/drivers/chqflag.cpp#L299
Not sure if your handling makes it to work properly anyway so the callback can be safely removed or not, iirc it was for night stages later on in that game.